runtime-common/current-run.ts

```diff
@@ -355,7 +387,7 @@ export class CurrentRun {
         loader: this.#loader,
       });
       cardType = Reflect.getPrototypeOf(card)?.constructor as typeof Card;
-      await api.recompute(card);
+      await api.recompute(card, { loadFields: true });
```

Comment:
in the search doc, if a linksTo field is notlaoded just include the ID. default loading fields in the index